src/Console/src/Sequence/CallableSequence.php

```diff
@@ -23,23 +23,15 @@ final class CallableSequence extends AbstractSequence
     /** @var string */
     private $function;
 
-    /** @var array */
-    private $parameters = [];
-
     /**
      * @param callable $function
-     * @param array    $parameters
-     * @param string   $header
-     * @param string   $footer
      */
     public function __construct(
         $function,
-        array $parameters = [],
```

Comment:
The private `$parameters` property actually never used, but remove parameter in the middle seems make a bug, I will revert and skip `RemoveUnusedPrivatePropertyRector`

Reply:
reverted and skipped `RemoveUnusedPrivatePropertyRector`